### PR TITLE
Firefox 78.1 add linux tests

### PR DESCRIPTION
**Pull request by bollwyvl**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Notes:
- backports changes from #26

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -17,19 +17,31 @@ source:
   sha256: 8765dfa132caf00a4ba6a71b7697d1d278ee9f6ae60a215be2f47dc081617064  # [win]
 
 build:
-  number: 0
+  number: 1
```
Comment (bollwyvl) [inline recipe/meta.yaml:20]: just commenting to verify i bumped this...

**Comment by birdsarah** [comment]
I'm confused. We just merged the PR into master. Why does this have many of the same changes? (I get the other PR which is against the :68 branch). I'm just it's something obvious, would just appreciate illumination.

**Comment by bollwyvl** [comment]
You're right, absolutely right, this was targeting the wrong branch!

**Comment by bollwyvl** [comment]
My goal is to support ESR releases as long as they are being used (selfishly, what I'm usually testing against). We'll keep getting 68 releases for some time... there might be a way to automate this, but I haven't explored it...

**Pull request merged**